tests/chez/chez021/expected

```diff
@@ -12,7 +12,7 @@ False
 4294967295
 18446744073709551615
 1/1: Building Bits (Bits.idr)
-Main> ["257", "200", "248", "1", "255", "200", "254"]
+Main> ["257", "200", "-8", "1", "-1", "200", "254"]
 Main> ["True", "True"]
-Main> ["255", "65535", "4294967295", "18446744073709551615"]
+Main> ["-1", "-1", "-1", "-1"]
```

Comment:
I think the expected values were correct before, as `Bits8` (through `Bits64`) are supposed to be unsigned integers. If you change the `castBitsNN` functions, the test output should match the previously expected values.

Reply:
Yes, these are here to test that the results of operations on the Bits primitives are still valid Bits values.